src/session_pool/session.js

```diff
@@ -0,0 +1,171 @@
+import EventEmitter from 'events';
+
+import { cryptoRandomObjectId } from 'apify-shared/utilities';
+import moment from 'moment';
+
+import { checkParamOrThrow } from 'apify-client/build/utils';
+import EVENTS from './events';
+
+
+/**
+ *  Class aggregating data for `Session`.
+ *  Session internal state can be enriched with custom user data for example some authorization tokens.
+ */
+export default class Session {
+    constructor(options = {}) {
+        const {
+            id = cryptoRandomObjectId(),
+            cookies = [],
```

Comment:
Shouldn't cookies be a part of userData? To be more generic and keep only variables that are needed for session management in properties?

Reply:
At first I thought that `userData` will be used only to seed `request.userData`, but the docs say that it can include `headers` so now I'm a bit confused too. Although I feel that `cookies` are such an important part of the `session` that they deserve their own property.